### PR TITLE
Fix some blessed details

### PR DESCRIPTION
**Pull request by arimet**
- [x] Displays the list of marbles taken by a player correctly
- [x] change the layout alignments

### CODE REVIEW

File: api/src/app.service.ts [hunk 1]
```diff
@@ -54,25 +56,28 @@ export class AppService {
...
-    newGameState.graph = newGraph;
-    newGameState.currentPlayer = switchToNextPlayer(
-      gameState.currentPlayer,
-      gameState.players,
-    );
+    if (marbleWon > -1) {
+      gameState.players[gameState.currentPlayerId - 1].marblesWon.push(
+        marbleWon,
+      );
+      console.log(gameState.players);
```
Comment (djhi) [inline api/src/app.service.ts:70]: Debug spotted

**Review by djhi — CHANGES_REQUESTED** [review]

File: api/src/game.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const createNewGameState = (): GameState => {
 
   gameState.graph = graph;
   gameState.players = players;
-  gameState.currentPlayer = players[0];
+  gameState.currentPlayerId = players[0].playerNumber;
```
Comment (Clebiez) [inline api/src/game.ts:34]: To improve readability, try to use the same language between resources.
`players[0].playerNumber` is an ID ? So call it `players[0].id`
Or maybe `gameState.currentPlayerId` refer just to the `playerNumber` ? so call it `gameState.currentPlayerNumber`

File: api/src/game.ts [hunk 3]
```diff
@@ -56,13 +56,17 @@ const initializePlayers = () => {
...
 
 export const switchToNextPlayer = (
-  actualPlayer: Player,
-  players: Array<Player>,
-): Player => {
-  if (actualPlayer.playerNumber === 1) return players[1];
-  return players[0];
+  actualPlayerID: number,
+): number => {
+  if (actualPlayerID === 1) return 2;
```
Comment (Clebiez) [inline api/src/game.ts:61]: In the game state you used `currentPlayerId` so in this case you should name this params `currentPlayerId`

File: client/src/api.ts [hunk 4]
```diff
@@ -133,8 +134,12 @@ export const pullActions = async (
     );
 
     if (canMoveMarble) {
-      const newGameState = await moveMarble(gameState, direction, player);
-      renderScreenView(newGameState);
+      try {
+        const newGameState = await moveMarble(gameState, direction, player);
+        renderScreenView(newGameState);
+      } catch (e) {
+        console.log("Can't moe this marble");
```
Comment (Clebiez) [inline client/src/api.ts:141]: Typo
Also instead of using console.log you should use console.error if possible

**Review by djhi — APPROVED** [review]

**Review by Clebiez — CHANGES_REQUESTED** [review]
The first things to have a project readable across developers is to speak the same language.
A good variables & functions naming is a key to achieve this !

Variable should explain what it contains, and if it's not possible you have to add a comment.
And variables with same content should be consistent in the whole project.

**Pull request merged**